crux-llvm/tests/abduction/README.md

```diff
@@ -0,0 +1,52 @@
+### The Goal 
```

Comment:
Now that we have abduction-related test cases under `test-data/golden`, is there any reason to keep this `tests/abduction` subdirectory around? It looks like some of the test cases from this subdirectory have been moved over to the golden tests, but not all of them. Is this something that you plan to do, or are the other test cases no longer relevant?

Reply:
I was going to remove this subdirectory once we got the tests in `test-data/golden` to pass. This is a bigger set of tests, many of which timeout when asked to find 3 abducts. We'll use this (among hopefully a bigger set) to improve the feature. I expect more of these to become tractable as we play with grammars, but for now, I only added a few tests that run quickly to the test suite.